Makefile

```diff
@@ -13,20 +13,19 @@ package-lock.json: package.json
 
 node_modules: package.json package-lock.json
 
-cljtest:
-	clojure -M:cljtest
+test-clj:
```

Comment:
Looks like this breaks symmetry with flure.db:

```
[db] [main $]
% >>> make cljtest                                           zonotope@gertrude
clojure -M:cljtest
Checking out: https://github.com/fluree/json-ld.git at 2e69362f611def8cc6793c6ddf24e4671fb6f397
...

------------------------------------------------------
[db] [main $]
% >>> make test-clj                                          zonotope@gertrude
make: *** No rule to make target 'test-clj'.  Stop.
```

Reply:
...which I'm personally fine with if we like this name better, but we should change db's names too to match. And then maybe leave targets in there for now w/ the old names that just echo e.g. "You want `make test-whatever now`" to nag me into updating my muscle memory. :)

Reply:
I was following the convention in the json-ld library, so it looks like the harmonizing needs to go a bit further. I guess I like the <action>-<target> formulation but it's pretty arbitrary.

Reply:
Alright, standardized on cljtest and cljstest :+1: